src/groupeddataframe/splitapplycombine.jl

```diff
@@ -496,6 +496,16 @@ wrap_table(x::Union{NamedTuple{<:Any, <:Tuple{Vararg{AbstractVector}}},
                     AbstractVector,
                     AbstractMatrix}) = wrap(x)
 
+wrap_row(x::Any) = wrap(x)
```

Comment:
As for `wrap_table` I'd say it's better to have the `::Any` fallback throw an error (for safety), and define specific methods that call `wrap` for `DataFrameRow` and `NamedTuple`.

Reply:
This is not possible I think, as we accept anything but a list I exclude in this code as scalars, see e.g. (this is run on 0.20.2):
```
julia> df = DataFrame(g=[1,2,3], x = [rand(1,1,1), (1,2,3), Ref(1)])
3×2 DataFrame
│ Row │ g     │ x                  │
│     │ Int64 │ Any                │
├─────┼───────┼────────────────────┤
│ 1   │ 1     │ [0.642618]         │
│ 2   │ 2     │ (1, 2, 3)          │
│ 3   │ 3     │ RefValue{Int64}(1) │

julia> by(df, :g, :x => x -> x[1])
3×2 DataFrame
│ Row │ g     │ x_function         │
│     │ Int64 │ Any                │
├─────┼───────┼────────────────────┤
│ 1   │ 1     │ [0.642618]         │
│ 2   │ 2     │ (1, 2, 3)          │
│ 3   │ 3     │ RefValue{Int64}(1) │
```

so by default we accept anything except this is something that is considered a multi-row data.

Reply:
Ah right, the old code checked after wrapping.